public/app/util/flamebearer.ts

```diff
@@ -116,6 +116,7 @@ export function flamebearerToDataFrameDTO(
     case 'samples':
     case 'trace_samples':
     case 'lock_nanoseconds':
+    case 'nanoseconds':
```

Comment:
Same remark as above, it seems we have too many ways to refer to a profile metric unit.